lib/OpenQA/CacheService/Model/Cache.pm

```diff
@@ -160,8 +159,7 @@ sub track_asset ($self, $asset) {
     eval {
         my $db = $self->sqlite->db;
         my $tx = $db->begin('exclusive');
-        my $sql = "INSERT INTO assets (filename, size, last_use) VALUES (?, 0, strftime('%s','now'))"
-          . "ON CONFLICT (filename) DO UPDATE SET pending=1";
+        my $sql = "INSERT INTO assets (filename, size, last_use) VALUES (?, 0, strftime('%s','now'))" . "ON CONFLICT (filename) DO UPDATE SET pending=1";
```

Comment:
I guess we could live with that, but ideally we would manually remove the `" . "` for such cases. But that could be a LOT :)

Reply:
Did a quick lookup, not so many, 8811a7ac1 tackles them.